ksql-parser/src/main/java/io/confluent/ksql/parser/SqlFormatter.java

```diff
@@ -347,8 +347,11 @@ protected Void visitCreateStream(CreateStream node, Integer indent) {
               .append(" ")
               .append(tableElement.getType());
         }
-        builder.append(")").append(" WITH (");
-        addComma = false;
+        builder.append(")");
+      }
+      if (!node.getProperties().isEmpty()) {
```

Comment:
Is this a regression or a fix for an existing bug?

Reply:
This is a fix for an existing bug.